From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Fedora Python maintainers <python-devel@lists.fedoraproject.org>
Date: Wed, 15 Jul 2020 15:19:19 +0200
Subject: [PATCH] 00104-lib64-fix-for-test_install.patch

00104 #
Only used when "%%{_lib}" == "lib64"
Another lib64 fix, for distutils/tests/test_install.py; not upstream:
---
 Lib/distutils/tests/test_install.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Lib/distutils/tests/test_install.py b/Lib/distutils/tests/test_install.py
index 397e2a7d4bb..c72ae0b6cdb 100644
--- a/Lib/distutils/tests/test_install.py
+++ b/Lib/distutils/tests/test_install.py
@@ -58,8 +58,9 @@ class InstallTestCase(support.TempdirManager,
             self.assertEqual(got, expected)
 
         libdir = os.path.join(destination, "lib", "python")
+        platlibdir = os.path.join(destination, "lib64", "python")
         check_path(cmd.install_lib, libdir)
-        check_path(cmd.install_platlib, libdir)
+        check_path(cmd.install_platlib, platlibdir)
         check_path(cmd.install_purelib, libdir)
         check_path(cmd.install_headers,
                    os.path.join(destination, "include", "python", "foopkg"))
